testing/mock/mock.go

```diff
@@ -150,3 +150,10 @@ func (KeyPath) String() string {
 func (KeyPath) Empty() bool {
 	return false
 }
+
+var _ exported.Height = Height{}
+
+// Height defines a placeholder struct which implements the exported.Height interface
+type Height struct {
+	exported.Height
```

Comment:
embedding this interface means that `Height` will have a bunch of no-op implementations of the interface methods?

Reply:
I didn't know you could do this 😄

Reply:
it will have whatever impl the concrete `exported.Height` its initialized with has, empty init leads to a `nil` being there which blows up if you try and call methods. (which is ok, we only want conformance to interface)